### DISCUSSION TITLE
Welcome to fingerpaint Discussions!

### DISCUSSION THREAD

**Discussion by Wazzaps** [Wazzaps/fingerpaint#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jhossbach** [comment]
Hey there, it might be a good idea to create a version tag / release for the package. I can pick up from there and create an AUR package.

**Reply by Wazzaps** [reply]
I just released a new version, will the AUR pkgbuild update automatically?

**Reply by jhossbach** [reply]
The PKGBUILD needs to be updated manually, but I can do that. For wayland, `xinput` is not needed as dependency, right? I will create an extra AUR package for wayland without xinout as dependency.

**Reply by jhossbach** [reply]
I just created the AUR package, see https://aur.archlinux.org/packages/fingerpaint-wayland

**Reply by Wazzaps** [reply]
Awesome :)

**Reply by jhossbach** [reply]
Can you also add the fingerpaint-wayland AUR package in the Readme?
